app/models/media_attribute.rb

```diff
@@ -0,0 +1,34 @@
+# == Schema Information
+#
+# Table name: media_attribute
+#
+#  id            :integer          not null, primary key
+#  high_title    :string           not null
+#  low_title     :string           not null
+#  neutral_title :string           not null
+#  slug          :string           not null, indexed
+#  title         :string           not null, indexed
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_media_attribute_on_slug   (slug)
+#  index_media_attribute_on_title  (title)
+#
+
+class MediaAttribute < ActiveRecord::Base
+  self.table_name = 'media_attribute'
+  extend FriendlyId
+  friendly_id :title, use: %i[slugged finders history]
```

Comment:
Is there a reason for this to have a slug?  Is there any place where we put the media attribute in a user-visible URL?